backend/ibutsu_server/scripts/mongo2postgres.py

```diff
@@ -69,27 +72,23 @@
 FIELDS_TO_TYPECAST = ["navigable", "weight"]
 
 # json indexes for the tables
-INDEXES = {
-    "results": [
-        "CREATE INDEX IF NOT EXISTS ix_results_jenkins_job_name "
-        "ON results((data->'jenkins'->>'job_name'));",
-        "CREATE INDEX IF NOT EXISTS ix_results_jenkins_build_number "
-        "ON results((data->'jenkins'->>'build_number'));",
-        "CREATE INDEX IF NOT EXISTS ix_results_classification "
-        "ON results((data->>'classification'));",
-        "CREATE INDEX IF NOT EXISTS ix_results_assignee " "ON results((data->>'assignee'));",
-        "CREATE INDEX IF NOT EXISTS ix_results_exception_name "
-        "ON results((data->>'exception_name'));",
-    ],
-    "runs": [
-        "CREATE INDEX IF NOT EXISTS ix_runs_jenkins_job_name "
-        "ON runs((data->'jenkins'->>'job_name'));",
-        "CREATE INDEX IF NOT EXISTS ix_runs_jenkins_build_number "
-        "ON runs((data->'jenkins'->>'build_number'));",
-        "CREATE INDEX IF NOT EXISTS ix_runs_jjn_jbn "
-        "ON runs((data->'jenkins'->>'build_number'), (data->'jenkins'->>'job_name'))",
-    ],
-}
+INDEXES = [
+    "CREATE INDEX IF NOT EXISTS ix_results_jenkins_job_name "
+    "ON results((data->'jenkins'->>'job_name'));",
+    "CREATE INDEX IF NOT EXISTS ix_results_jenkins_build_number "
+    "ON results((data->'jenkins'->>'build_number'));",
+    "CREATE INDEX IF NOT EXISTS ix_results_classification "
+    "ON results((data->>'classification'));",
+    "CREATE INDEX IF NOT EXISTS ix_results_assignee " "ON results((data->>'assignee'));",
+    "CREATE INDEX IF NOT EXISTS ix_results_exception_name "
+    "ON results((data->>'exception_name'));",
+    "CREATE INDEX IF NOT EXISTS ix_runs_jenkins_job_name "
+    "ON runs((data->'jenkins'->>'job_name'));",
+    "CREATE INDEX IF NOT EXISTS ix_runs_jenkins_build_number "
+    "ON runs((data->'jenkins'->>'build_number'));",
+    "CREATE INDEX IF NOT EXISTS ix_runs_jjn_jbn "
+    "ON runs((data->'jenkins'->>'build_number'), (data->'jenkins'->>'job_name'))",
```

Comment:
We should perhaps add one additional index for the runs:
```
CREATE INDEX IF NOT EXISTS ix_runs_summary ON runs USING gin (summary)
```

Reply:
Sure! I'll add that.